styleguide.config.js

```diff
@@ -36,14 +37,16 @@ module.exports = {
         path.join(__dirname, 'styleguide.setup.css'),
         path.join(__dirname, 'src/lib/style/main.css'),
     ],
-    // This works when running the local server, but not when built.
-    // styles: {
-    //     StyleGuide: {
-    //         content: {
-    //             maxWidth: 1300
-    //         }
-    //     }
-    // },
+    theme: {
+        maxWidth: 1300,
```

Comment:
found out how to properly customize the maxWidth (of the content)!

Reply:
Oops! Didn't see this. Do you need to add a unit after 1300?